test/integration/scene-manager-component.spec.ts

```diff
@@ -0,0 +1,206 @@
+  const mockWorldStreamingPermission = jest.fn()
+  const mockLandPermission = jest.fn()
+  const mockIsAdmin = jest.fn()
+
+  const mockWorld = {
+    fetchWorldActionPermissions: jest.fn(),
+    hasWorldOwnerPermission: mockWorldOwnerPermission,
+    hasWorldStreamingPermission: mockWorldStreamingPermission
+  }
+
+  const mockSceneAdminManager = {
+    isAdmin: mockIsAdmin,
+    addAdmin: jest.fn(),
+    removeAdmin: jest.fn(),
+    listActiveAdmins: jest.fn()
+  }
+
+  const mockLand = {
+    hasLandPermission: mockLandPermission
+  }
+
+  let sceneManager: ISceneManager
+
+  beforeEach(async () => {
+    jest.clearAllMocks()
+
+    sceneManager = await createSceneManagerComponent({
+      land: mockLand,
+      world: mockWorld as any,
+      sceneAdminManager: mockSceneAdminManager as any
+    })
+  })
+
+  const mockScenePlace: PlaceAttributes = {
+    id: 'place-123',
+    title: 'Test Place',
+    description: null,
+    image: null,
+    highlighted_image: null,
+    owner: '0xowner',
+    positions: ['0,0'],
+    base_position: '0,0',
+    contact_name: null,
+    contact_email: null,
+    likes: 0,
+    dislikes: 0,
+    favorites: 0,
+    like_rate: null,
+    like_score: null,
+    highlighted: false,
+    disabled: false,
+    disabled_at: null,
+    created_at: new Date(),
+    updated_at: new Date(),
+    world: false,
+    world_name: null,
+    deployed_at: new Date(),
+    categories: [],
+    user_like: false,
+    user_dislike: false,
+    user_favorite: false,
+    user_count: 0,
+    user_visits: 0
+  }
+
+  const mockWorldPlace: PlaceAttributes = {
+    id: 'world-123',
+    title: 'Test World',
+    description: null,
+    image: null,
+    highlighted_image: null,
+    owner: '0xowner',
+    positions: [],
+    base_position: '0,0',
+    contact_name: null,
+    contact_email: null,
+    likes: 0,
+    dislikes: 0,
+    favorites: 0,
+    like_rate: null,
+    like_score: null,
+    highlighted: false,
+    disabled: false,
+    disabled_at: null,
+    created_at: new Date(),
+    updated_at: new Date(),
+    world: true,
+    world_name: 'test-world.eth',
+    deployed_at: new Date(),
+    categories: [],
+    user_like: false,
+    user_dislike: false,
+    user_favorite: false,
+    user_count: 0,
+    user_visits: 0
+  }
+
+  describe('isSceneOwner', () => {
+    it('should call hasWorldOwnerPermission for a world', async () => {
+      mockWorldOwnerPermission.mockResolvedValue(true)
+
+      const result = await sceneManager.isSceneOwner(mockWorldPlace, '0xuser')
+
+      expect(mockWorldOwnerPermission).toHaveBeenCalledWith('0xuser', 'test-world.eth')
+      expect(result).toBe(true)
+    })
+
+    it('should call hasLandPermission for a scene', async () => {
+      mockLandPermission.mockResolvedValue(true)
+
+      const result = await sceneManager.isSceneOwner(mockScenePlace, '0xuser')
+
+      expect(mockLandPermission).toHaveBeenCalledWith('0xuser', ['0,0'])
+      expect(result).toBe(true)
+    })
```

Comment:
There's no need to test if these functions where called, we can already check these paths with the tests below. What do you think about removing them?